scripts/build-module.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/node
+
+'use strict'
+
+const path = require('path')
+const glob = require('glob')
+const fs = require('fs')
+
+const babel = require('babel-core')
+const outputFileSync = require('output-file-sync')
+
+let options = JSON.parse(require('fs').readFileSync('.babelrc', 'utf8'))
```

Comment:
look here, i don’t duplicate them!

instead i read the .babelrc and replace one preset with another programmatically!